lib/foods.js

```diff
@@ -1,65 +1,75 @@
 $('document').ready(function() {
+  for (var i = 0; i < localStorage.length; i++){
```

Comment:
since this doesn't rely on anything specific within the document ready function, i'd suggest breaking this out into its own function 👍